M2/Macaulay2/packages/VectorGraphics.m2

```diff
@@ -6,7 +6,7 @@ newPackage(
         Authors => {{Name => "Paul Zinn-Justin",
                   Email => "pzinn@unimelb.edu.au",
                   HomePage => "http://blogs.unimelb.edu.au/paul-zinn-justin/"}},
-        Headline => "A package to produce SVG graphics",
+        Headline => "producing SVG graphics",
```

Comment:
I'm trying to make all package headlines uniform, which includes removing redundant clauses. @pzinn is this change okay with you?